### ISSUE TITLE
error on ddays(5.1) 

### ISSUE THREAD

**Issue by xhdong-umd** [hadley/lubridate#510]
```
> ddays(3.2)
[1] "276480s (~3.2 days)"
> ddays(8.15)
[1] "704160s (~1.16 weeks)"
> ddays(5.1)
Error in sprintf("%ds (~%s %ss)", x, x2, unit, "s)") : 
  invalid format '%d'; use format %f, %e, %g or %a for numeric objects
> ddays(5.10)
Error in sprintf("%ds (~%s %ss)", x, x2, unit, "s)") : 
  invalid format '%d'; use format %f, %e, %g or %a for numeric objects
> dminutes(5.1)
[1] "306s (~5.1 minutes)"
```

Why is 5.1 for ddays so special?

```
> sessionInfo()
R version 3.3.2 (2016-10-31)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: macOS Sierra 10.12.2

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] lubridate_1.6.0

loaded via a namespace (and not attached):
[1] magrittr_1.5  tools_3.3.2   stringi_1.1.2 stringr_1.1.0
```

**Comment by cderv** [comment]
With last dev version of lubridate, for me, it is working. 
``` r
library(lubridate)
#> 
#> Attachement du package : 'lubridate'
#> The following object is masked from 'package:base':
#> 
#>     date

ddays(3.2)
#> [1] "276480s (~3.2 days)"
ddays(8.15)
#> [1] "704160s (~1.16 weeks)"
ddays(5.1)
#> [1] "440640s (~5.1 days)"
ddays(5.10)
#> [1] "440640s (~5.1 days)"
dminutes(5.1)
#> [1] "306s (~5.1 minutes)"
```

<details><summary>Session info</summary>

``` r
devtools::session_info()
#> Session info -------------------------------------------------------------
#>  setting  value                       
#>  version  R version 3.3.2 (2016-10-31)
#>  system   x86_64, mingw32             
#>  ui       RTerm                       
#>  language (EN)                        
#>  collate  French_France.1252          
#>  tz       Europe/Paris                
#>  date     2017-01-19
#> Packages -----------------------------------------------------------------
#>  package   * version     date       source                            
#>  backports   1.0.4       2016-10-24 CRAN (R 3.3.2)                    
#>  devtools    1.12.0.9000 2017-01-08 Github (hadley/devtools@1ce84b0)  
#>  digest      0.6.11      2017-01-03 CRAN (R 3.3.2)                    
#>  evaluate    0.10        2016-10-11 CRAN (R 3.3.2)                    
#>  htmltools   0.3.5       2016-03-21 CRAN (R 3.3.2)                    
#>  knitr       1.15.1      2016-11-22 CRAN (R 3.3.2)                    
#>  lubridate * 1.6.0.9009  2017-01-19 Github (hadley/lubridate@ebd90d9) 
#>  magrittr    1.5         2014-11-22 CRAN (R 3.3.2)                    
#>  memoise     1.0.0       2016-01-29 CRAN (R 3.3.2)                    
#>  pkgbuild    0.0.0.9000  2017-01-08 Github (r-pkgs/pkgbuild@65eace0)  
#>  pkgload     0.0.0.9000  2017-01-08 Github (r-pkgs/pkgload@def2b10)   
#>  Rcpp        0.12.8      2016-11-17 CRAN (R 3.3.2)                    
#>  rmarkdown   1.3         2017-01-14 Github (rstudio/rmarkdown@3544f1d)
#>  rprojroot   1.1         2016-10-29 CRAN (R 3.3.2)                    
#>  stringi     1.1.2       2016-10-01 CRAN (R 3.3.2)                    
#>  stringr     1.1.0       2016-08-19 CRAN (R 3.3.2)                    
#>  withr       1.0.2       2016-06-20 CRAN (R 3.3.2)                    
#>  yaml        2.1.14      2016-11-12 CRAN (R 3.3.2)
```

</details>

**Comment by xhdong-umd** [comment]
OK that's good. 

One question though, is this a known bug so the development version fixed, or it's "fixed" by some unknown reason, so the bug could still exist for other value, or cause other problem?

**Comment by vspinu** [comment]
I think it was fixed by #465. This is a printing error.

**Comment by xhdong-umd** [comment]
OK That's good to know. Thanks!
